src/main/kotlin/org/blueliner/springbootwithkotlin/service/JokeService.kt

```diff
@@ -0,0 +1,59 @@
+package org.blueliner.springbootwithkotlin.service
+
+import org.blueliner.springbootwithkotlin.dto.JokeDto
+import org.springframework.ai.chat.client.ChatClient
+import org.springframework.ai.chat.client.ChatClient.PromptUserSpec
+import org.springframework.ai.chat.messages.Message
+import org.springframework.ai.chat.messages.UserMessage
+import org.springframework.ai.chat.model.ChatResponse
+import org.springframework.ai.chat.prompt.Prompt
+import org.springframework.ai.chat.prompt.SystemPromptTemplate
+import org.springframework.ai.openai.OpenAiChatModel
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.core.ParameterizedTypeReference
+import org.springframework.core.io.Resource
+import org.springframework.stereotype.Service
+
+
+@Service
+class JokeService(
+    private val chat: OpenAiChatModel,
+) {
+    @Value("classpath:/prompts/system-message.st")
+    private val systemResource: Resource? = null
+
+
+    fun getJokeUsingBasicAiModelOutput(): ChatResponse {
```

Comment:
nit: its better to return specific return types, chatresponse is too vague. when you see the method called "getJokeUsingBasicAiModelOutput" I assume to get "String" as return, not ChatResponse